snapcraft/repo/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Package repository helpers."""
+
+from .apt_key_manager import AptKeyManager  # noqa: F401
+from .apt_sources_manager import AptSourcesManager  # noqa: F401
+from .package_repository import PackageRepository  # noqa: F401
+from .package_repository import PackageRepositoryApt  # noqa: F401
+from .package_repository import PackageRepositoryAptPPA  # noqa: F401
```

Comment:
instead of adding noqa consider declaring __all__ for the package

Reply:
Done. The decoupling change also drastically reduced the amount of exported symbols, but having `__all__` won't hurt (and it will be needed if we spin off craft-repositories).